codecov_auth/admin.py

```diff
@@ -310,6 +311,66 @@ class AccountAdmin(AdminMixin, admin.ModelAdmin):
         "is_delinquent",
     ]
 
+    @admin.action(description="Link Users to Account")
+    def link_users_to_account(self, request, queryset):
+        for account in queryset:
+            all_plan_activated_user_ids = set()
+            for org in account.organizations.all():
+                all_plan_activated_user_ids.update(set(org.plan_activated_users))
+
+            all_plan_activated_users = Owner.objects.filter(
+                ownerid__in=all_plan_activated_user_ids
+            ).prefetch_related("user")
+            non_student_count = all_plan_activated_users.exclude(student=True).count()
+            total_seats_for_account = account.plan_seat_count + account.free_seat_count
+            if non_student_count > total_seats_for_account:
+                self.message_user(
+                    request,
+                    f"Request failed: Account plan does not have enough seats; "
+                    f"current plan activated users (non-students): {non_student_count}, total seats for account: {total_seats_for_account}",
+                    messages.ERROR,
+                )
+                return
+
+            owners_without_user_objects = all_plan_activated_users.filter(
+                user__isnull=True
+            )
+            userless_owners = []
+            for userless_owner in owners_without_user_objects:
+                new_user = User.objects.create()
```

Comment:
Is this new User object meant to be empty?

Reply:
no! fixed!
ty ty ty